src/js/components/Alert/Alert.tsx

```diff
@@ -1,26 +1,51 @@
 import React from 'react';
 import classNames from 'classnames';
+import Icon from '../Icon/Icon';
+import { IconNameType } from '../Icon/icons';
 
 type Props = {
   className?: string;
+  header?: string;
+  icon?: IconNameType;
   tag?: keyof JSX.IntrinsicElements | typeof React.Component;
   theme?: string;
+  hideIcon?: boolean;
   children: React.ReactNode;
   [key: string]: any;
 };
 
 const Alert: React.FC<Props> = ({
   children,
   className,
+  header,
+  icon,
   tag: Tag = 'div',
   theme = 'primary',
+  hideIcon = false,
```

Comment:
By default, we show the icon as requested by design but if no icon is set we won't have any empty element as explained earlier